### DISCUSSION TITLE
Using motion detection with https internally.

### DISCUSSION THREAD

**Discussion by GitKiller-HA** [fwestenberg/reolink_dev#267] [Ideas]
Hi,

When there will be possible using motion detection with https internally. In my opinion almost every one using https so it would be very nice to get this option.

Thank You,

Grzegorz

**Comment by andriej** [comment]
It depends on Reolink firmware update AFAIR, which is still not released nor announced :-/

**Comment by GitKiller-HA** [comment]
Thank you for replay.
Do you know if someone asked Reolink is going to implement this in nearest future ?

**Reply by andriej** [reply]
I guess it was @fwestenberg

**Comment by GitKiller-HA** [comment]
Hi @fwestenberg 
Could you tell us what's the state is ?
May be I can help somehow ?

**Comment by timmyag** [comment]
If there is something we need to request of Reolink let us know what it is.

**Reply by cpainchaud** [reply]
Yes, ask them to support HTTPS webhooks. Many people asked already, me included

**Comment by GitKiller-HA** [comment]
Hi,

I wrote to Reolink support emails to know something, as below (main body). It doesn't look like it would be fast :(

**Me:**
A lot of us using your products with Home Assistant. For better security we use https connection even in local network. This configuration couse problem with motion sensor in camera. Problem is due to that current firmware does not support HTTPS based webhooks.
Could you declare when are you going to add this functionality ?
Please note that this is very important for all users who want to keep their solutions safe.

**Reolink replay:**
Thank you for choosing Reolink!
May I know do you mean when using the camera with the home assistant, the motion detection won't work?
As far as I know, the camera only supports live view on smart home devices.
We have been diligently working on implementing some selected user requests. And I’m honored to have yours added to the list. Some nice ideas have been implemented as you can see in our changelog (What’s New) of firmware/software updates. Please kindly understand that bug fixes are our top priority and then user requests or feature requests. So it may take some time to see yours become true. Please stay tuned!
Thanks for your support of the Reolink Product!
Anything we can help you with, please do not hesitate to keep us informed.
Have a nice day!

**Me replay:**
There is a great support (in Home Assistant) for reolink cameras, not just live view. It is a great job (https://github.com/fwestenberg/reolink_dev). I realized that the bug fixes are the most important but security functionalities (IMHO) are much more (paradox) important (I am sure I mustn't  say why). You should contact the author of reolink integration and make something more for your customers. Home Assistant is the fastest developing software for support intelligent home/house. Every user of HA checks, before buying, if appliances or hardwares are supported by HA. I didn't do this .... :(
So, if you want your products to be used in more range systems including DIY solutions you should consider implementing ssl via webhooks much faster.

**Reolink replay:**
Thank you very much for your suggestion of SSL via webhooks, we will forward it to our R&D team.
As you know,technological breakthroughs require time and the efforts of R&D personnel. It will take time to evaluate the request and to implement it.
You may subscribe to our emails to get the news: https://reolink.com/#footer.

**Comment by Padm59** [comment]
Any new information on this topic?

**Comment by macxq** [comment]
Hi all, let’s speak the same voice on the Reolink community page. I have started a topic there     https://community.reolink.com/topic/3291/third-party-integration-support-ai-detection-home-assistant  please share your opinion there 🗣

**Comment by macxq** [comment]
I don't think we'll ever get https support and proper AI event propagation :/  
https://community.reolink.com/topic/3291/third-party-integration-support-ai-detection-home-assistant/6?_=1652855343393 don't think that
